agg-ast/ast/src/definitions.rs

```diff
@@ -22,7 +22,7 @@ use serde::{Deserialize, Serialize};
 /// Stage represents an aggregation pipeline stage.
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub enum Stage {
```

Comment:
collection isn't actually used by mongodb, but there is no reason not to support it.

Reply:
We must have used this in the desugarer at some point and just kept it around all this time. The desugarer does not currently use this so it's unclear what it's original purpose was.

Reply:
I think it's supported on ADF?